CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import UIKit
+import EventKit
 
 struct EventCellParameters {
```

Comment:
Should we use this struct or the Event class to hold the data? Seems a bit redundant right now